rhizome/postgres/bin/configure

```diff
@@ -48,25 +48,27 @@ pg_hba_entries = <<-PG_HBA
 # Database administrative login by Unix domain socket
 local   all             postgres                                peer map=system2postgres
 local   all             pgbouncer                               peer map=system2pgbouncer
+local   replication     all                                     peer
```

Comment:
It is bit misleading, but `all` does not cover `replication`. So no need to move this.

Second keyword in each row refers to database, so `all` covers all databases, but there is no database named `replication`. It is seen as pseudo-database.